.github/workflows/release-latest.yml

```diff
@@ -0,0 +1,93 @@
+name: npm - release to latest
+run-name: npm - release ${{ github.event.inputs.targetVersion }} from ${{ github.ref_name }} on latest tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      targetVersion:
+        description: 'Version to publish on latest tag (e.g. 5.0.0)'
+        type: string
+        required: true
+      skipEcosystemTestsChecks:
+        description: 'Skip ecosystem tests checks? (e.g. true)'
+        type: boolean
+      skipPackages:
+        description: 'Skip some packages? (e.g. `@prisma/debug,@prisma/internals`)'
+        type: string
+        required: false
+      onlyPackages:
+        description: 'Only publish some packages? (e.g. `@prisma/debug,@prisma/internals`)'
+        type: string
+        required: false
```

Comment:
It would be nice to have a `dryRun` input here for the future if we want to debug or change it.